src/app/core/game-board/game-board.component.html

```diff
@@ -1,5 +1,5 @@
 <mat-card class="game-board-card" fxFill>
-  <div class="score-top">
-    <h1>Score Board</h1>
+  <div class="game-board-top">

```

Comment:
did the css get changed to match?

Reply:
I can't remember if its needed, if not ignore comment.

Reply:
I believe the css is fine.